bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -51,6 +51,8 @@ parser.add_argument('--extra-cpt-fraction', type=float,
                     help='Fraction of the extra component in the signal density')
 parser.add_argument('--temp-volume', type=float, 
                     help='Volume covered by the template bank')
+parser.add_argument('--mchirp-downsample-power', type=float, default=1.0,
```

Comment:
I think it would be better to have a default of None, then the code does not do the downsampling unless a value is given